docs/install/amazon-web-services-eks.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Amazon Web Services Elastic Kubernetes Service (AWS EKS)"
+linkTitle: "Amazon Web Services Elastic Kubernetes Service (AWS EKS)"
+---
+
+## Overview
+
+This guide will demonstrate how to install Ondat onto a [Elastic Kubernetes Service](https://aws.amazon.com/eks/) cluster using the [Ondat kubectl plugin](/docs/reference/kubectl-plugin/).
+
+## Prerequisites
+
+* You have met the minimum resource requirements for Ondat to successfully run. Review the main [Ondat prerequisites](/docs/prerequisites/) page for more information.
+
+* The following CLI utilities are installed on your local machine and are available in your `$PATH`:
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [kubectl-storageos](/docs/reference/kubectl-plugin/)
+
+* You have a running EKS cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
+
+* Your EKS clusters use [Ubuntu for EKS](https://cloud-images.ubuntu.com/docs/aws/eks/) as the default node operating system with an optimised kernel. For the kernel module `tcm_loop`, the package `linux-modules-extra-$(uname -r)` is additionally required on each of the nodes - this can be installed automatically by adding it to the node's user data as in the example below.
+
+In this example, we have used [eksctl](https://eksctl.io/introduction/) to create a cluster with 3 nodes of size `t3.large` running Ubuntu for EKS in the `eu-west-2` region. We have provided `100 GB` of disk space for each node - note that by default, Ondat will store data locally in the node's file system under the path `/var/lib/storageos` on each node in [hyperconverged mode](/docs/concepts/nodes/#hyperconverged-mode). In a production infrastructure, we would likely create multiple EBS Volumes tweaked for performance or use ephemeral SSD storage and [mount our volumes under data device directories](/docs/concepts/volumes/) with some additions to user data. We would also implement some form of snapshots or backup of these underlying volumes to ensure continuity in a disaster scenario.
+
+```yaml
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: ondat-cluster
+  region: eu-west-2
+
+managedNodeGroups:
+  - name: ondat-ng
+    minSize: 3
+    maxSize: 3
+    instanceType: t3.large
+    ami: ami-0cb2cb474d9e4e075
```

Comment:
eksctl lets you use the distro family to Ubuntu or AmazonLinux. IMO that is a much better approach because the ami hardcoded will fail when readers use this guide on a different region. The ami id will be different and it is not easy to find the exact image. 

The option is:
```
managedNodeGroups:
- amiFamily: Ubuntu2004
```

Reply:
I've avoided specifying amiFamily due to https://eksctl.io/usage/eks-managed-nodes/#feature-parity-with-unmanaged-nodegroups

You're right, though, I'll definitely add a bit about AMI discovery!

Reply:
Added a blurb on AMI discovery.